### ISSUE TITLE
How to install?

### ISSUE THREAD

**Issue by csaba504** [alex-0103/warmup4IE#1]
Hy,

I tired to install it, but I got an error when restart the HA.

`Error while setting up platform warmup4ie
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/homeassistant/helpers/entity_platform.py", line 128, in _async_setup_platform
    SLOW_SETUP_MAX_WAIT, loop=hass.loop)
  File "/usr/local/lib/python3.6/asyncio/tasks.py", line 358, in wait_for
    return fut.result()
  File "/usr/local/lib/python3.6/concurrent/futures/thread.py", line 56, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/climate/warmup4ie.py", line 85, in setup_platform
    [Warmup4IE(hass, name, user, password, location, room, target_temp)])
  File "/config/custom_components/climate/warmup4ie.py", line 108, in __init__
    import warmup4ie
ModuleNotFoundError: No module named 'warmup4ie'`

This is the file list:
```
core-ssh:/config/custom_components# tree
.
├── climate
│   ├── __pycache__
│   │   └── warmup4ie.cpython-36.pyc
│   └── warmup4ie.py
├── setup.py
└── warmup4ie
    ├── __init__.py
    └── warmup4ie.py

3 directories, 5 files
```

How can I setup this component?

**Comment by 8osman** [comment]
Mine  did this too. Until I restarted home assistant from the command line, rather that the restart within the program. 

I also struggled with the location information that this required. Before logging back into warmup website and rechecking what I'd entered under the room config there.  Overall a great addition to my home assistant setup now, enabling me to easily switch this into away mode when we leave the house.
